darts/models/block_rnn_model.py

```diff
@@ -0,0 +1,194 @@
+"""
```

Comment:
These changes here are not really changes, I simply renamed the old RNN model. I did add a couple of paragraphs to the docstrings, however, so I would be happy if you could check those.